pippo-controller-parent/pippo-controller/src/main/java/ro/pippo/controller/ControllerHandler.java

```diff
@@ -462,7 +462,7 @@ protected void handleDeclaredThrownException(Exception e, RouteContext routeCont
     /*
      * Cleans a complex content-type or accept header value by removing the
      * quality scores.
-     * <p/>
+     * <p>
```

Comment:
Why?

Reply:
otherwise javadoc tool fails. See https://stackoverflow.com/questions/5260368/which-tag-should-be-used-as-paragraph-separator-in-javadoc and https://stackoverflow.com/questions/5260368/which-tag-should-be-used-as-paragraph-separator-in-javadoc